app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -117,29 +117,30 @@ export function testsForProviderType(providerType) {
         { name: 'eth_coinbase', numberOfParameters: 0 },
         { name: 'eth_createAccessList', numberOfParameters: 2 },
         { name: 'eth_feeHistory', numberOfParameters: 3 },
-        { name: 'eth_getFilterChanges', numberOfParameters: 1 },
```

Comment:
Why were these removed in this PR? Shuffling code around shouldn't affect the tests.

Reply:
These are removed before [we reverted that move subscribe and filter middleware commit](https://github.com/MetaMask/metamask-extension/pull/18129), since those middleware were no longer separable from the network client and we were going to have to put in bunch of effort to test them and we [were separating that effort out](https://github.com/MetaMask/metamask-extension/issues/16899). But now that that was reverted we can add them back!

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/17652/commits/4e3911a00695766a0edef23a4958edc66779bb24